pages/dkp/kommander/2.2/DKP-insights/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: DKP Insights
+title: DKP Insights
+excerpt: Use DKP Insights to monitor and tune your environment
+menuWeight: 55
+beta: false
+enterprise: false
+techPreview: true
+---
+
+<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only. </p>
+
+We designed DKP Insights to assist Kubernetes Administrators with daily tasks, such as ensuring your clusters are secure, debugging, monitoring, and anticipating issues. DKP Insights collects logs, events, and metrics from the DKP Management cluster and managed or attached Kubernetes clusters. It then generates rules-based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
+
+DKP Insights works seamlessly within networked or air-gapped environments. It collects metrics, logs, and events from multiple data sources and uses rule-based heuristics to detect issues from the data and generate insight items for the users.
+
+You can filter the insight items for a selected cluster or project by:
```

Comment:
Hmm, I'm not sure this wording is 100% accurate either. 
The user can either filter by:
- the type categories listed below
- project name
- cluster name
- description 
and they can toggle into each severity level.